### ISSUE TITLE
Hi, There seems to be a problem in the first convolution layer of 'AutoEncoderNet' ?

### ISSUE THREAD

**Issue by heartxkl** [weberhen/learning_indoor_lighting#8]
Hi, In the currently released code,  The first convolution layer of 'AutoEncoderNet' is 
`nn.Conv2d(3, 64, 9, stride=1, padding=1)`
If I understand correctly, the shape of input  is 3 x 64 x 128 (CHW), so why the kernel size of  the first convolution layer  is 9 ?
It cannot output a 64 x 32 x 64(CHW) tensor.
Thanks for your reply in advance.

**Comment by weberhen** [comment]
Hi!

Sorry about that, just ignore the comment on those lines, I made different experiments and at some point stop updating the comments. Your understanding is correct.

**Comment by heartxkl** [comment]
Thanks~
